src/game/game.cpp

```diff
@@ -6447,24 +6443,40 @@ void Game::addDistanceEffect(const SpectatorHashSet& spectators, const Position&
 
 void Game::startDecay(Item* item)
```

Comment:
same here, this can go and all the logic can go to g_decay

Reply:
Done!